exercises/binary-search/example.cpp

```diff
@@ -0,0 +1,28 @@
+#include "binary_search.h"
+
+namespace binary_search {
+
+int find(std::vector<int> const& data, int value)
+{
+    int lpos = 0;
```

Comment:
These types are indexes so they should all be `size_t`

Reply:
Thanks. Changed this to size_t throughout.